src/service/SmsIngestionService/SmsIngestionController.ts

```diff
@@ -0,0 +1,16 @@
+import { APIClient, APIMapping } from '../../http';
+import {SmsIngestionServiceTypes} from "./SmsIngestionService.Types";
+
+export class SmsIngestionController extends APIClient {
+    constructor() {
+        super(APIMapping.smsIngestionService);
+    }
+
+    /**
+     * Sends the SMS message
+     * @param smsData
+     */
+    async fetchActivities(smsData: SmsIngestionServiceTypes.SmsData) {
+        return this.invokeApiWithErrorHandling('/send', 'POST', smsData);
```

Comment:
Please define a return type

```suggestion
        return this.invokeApiWithErrorHandling<?>('/send', 'POST', smsData);
```

Reply:
i didn't expected any specific type for response because it will be standard 200 OK, but added type string